ucs/clientlist.cpp

```diff
@@ -1097,8 +1136,7 @@ void Client::JoinChannels(std::string ChannelNameList) {
 	safe_delete(outapp);
 }
 
-void Client::LeaveChannels(std::string ChannelNameList) {
-
+void Client::LeaveChannels(std::string ChannelNameList, bool command_directed) {
```

Comment:
Should this be `const std::string&` to avoid making a copy? Also, `channel_name_list`?